research/flamby/fed_heart_disease/perfcl/README.md

```diff
@@ -0,0 +1,24 @@
+### Running hyperparameter sweep
+
+To run the hyperparameter sweep you simply run the command
+
+```bash
+./research/flamby/fed_heart_disease/perfcl/run_hp_sweep.sh \
+   path_to_config.yaml \
+   path_to_folder_for_artifacts/ \
+   path_to_folder_for_dataset/ \
+   path_to_desired_venv/
+```
+
+from the top level directory of the repository
+
+An example is something like
+``` bash
+./research/flamby/fed_heart_disease/perfcl/run_hp_sweep.sh \
+   research/flamby/fed_heart_disease/perfcl/config.yaml \
+   research/flamby/fed_heart_disease/perfcl/ \
+   /Users/david/Desktop/FLambyDatasets/fed_heart_disease/ \
+   /h/demerson/vector_repositories/fl4health_env/
+```
+
+In order to manipulate the grid search being conducted, you need to change the parameters for `lr`, the client-side learning rate, in the `run_hp_sweep.sh` script directly.
```

Comment:
Since there are two other hyper-parameters for PerFCL (mu and gamma), it's probably worth saying something like 
```
you need to change the parameters for `lr`, `mu`, or `gamma`, in the `run_hp_sweep.sh` script directly. These represent the client-side learning rate and the two PerFCL loss weights respectively.
```